scripts/yg-privacyenhanced.py

```diff
@@ -66,6 +67,10 @@ def create_my_orders(self):
                                              self.txfee * (1 + float(self.txfee_factor))))
         randomize_minsize = int(random.uniform(self.minsize * (1 - float(self.size_factor)),
                                                self.minsize * (1 + float(self.size_factor))))
+        if randomize_minsize < jm_single().DUST_THRESHOLD:
+            jlog.warn("Minsize was randomized to below dust; resetting to dust "
+                      "threshold: {} sats.".format(jm_single().DUST_THRESHOLD))
```

Comment:
Use `amount_to_str()` from `jmbitcoin` for formatting dust threshold value?

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/890/commits/b9f812967ca3c63a29938e6fd47b590443467e53